src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +57,43 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        if (dateParams.length == 0) {
+            return Optional.empty();
+        }
+        String date = "" + dateParams[0] + dateParams[1] + dateParams[2];
+        return getLocalDate(date, FORMATTER1);
```

Comment:
try to use LocalDate.of()

Reply:
I can't do it because if i use LocalDate.of()  i can't use getLocalDate(). And if i don't use this method I will need to catch DateTimeParseException in getDate() and it will be duplicate of method getLocalDate().